src/explorer/SiteTreeItem.ts

```diff
@@ -101,9 +102,7 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         this._logStream = kuduClient.getLogStream().on('data', chunk => {
             this._logStreamOutputChannel.append(chunk.toString());
         }).on('error', err => {
-            util.sendTelemetry('ConnectToLogStreamError', { name: err.name, message: err.message });
-            this._logStreamOutputChannel.appendLine('Error connecting to log-streaming service:');
-            this._logStreamOutputChannel.appendLine(err.message);
+            throw (err);
```

Comment:
I'm not sure how to test out this code (aka I don't know how to repro an error). @whiskeyjay can you help determine if this would be okay?

Mainly - I'm concerned that `.on('error', err => {` might send the error in chunks and the user might not see the whole error if we just throw the error immediately like this

Reply:
You can disconnect your network (or disable the network adapter from Device Manager) then an error will occur if the streaming has already started:

Connecting to log-streaming service...
2018-02-01T23:53:26  Welcome, you are now connected to log-streaming service.
2018-02-01T23:54:26  No new trace in the past 1 min(s).
Error connecting to log-streaming service:
read ECONNRESET
Disconnected from log-streaming service.

I don't think one single error will be sent in chunks.

Reply:
I suppose with "throw", errors will be shown in a message bar instead? When users connect to log stream, their eyes are on the output panel, usually at the bottom part of the window, looking for logs; why make them having to look across the screen for errors?

Reply:
Discussed with Jay-- the outputChannel for log-streaming is a different channel than where errors are normally displayed for App Service.  I think it's worth preserving the old experience.